app/grants/templates/grants/detail/info.html

```diff
@@ -81,7 +81,7 @@ <h1 class="font-title-xl my-4 font-weight-bold">
 
       <div class="mb-2">
         <i style="width: 14px;" class="fab fa-ethereum mr-2 text-center"></i>
-        <a class="wallet-address" href="https://etherscan.io/address/{{ grant.admin_address }}"
+        <a id="wallet-address" class="wallet-address" href="https://etherscan.io/address/{{ grant.admin_address }}"
```

Comment:
Do we want to have a rinkeby option here (rinkeby.etherscan.io)

Reply:
Ah i shalll fix that up ^_^